cobalt/build/configs/android-arm/args.gn

```diff
@@ -1,3 +1,7 @@
 target_os = "android"
 target_cpu = "arm"
+
+# TODO(b/378934120): Fix starboard compiler/linker errors while building with chromium flags
+clang_use_chrome_plugins = false
+enable_check_raw_ptr_fields = false
```

Comment:
I'd recommend leaving a one-liner comment that explains where this is declared and what does it do. It seems it enables a number of checks from a custom Chromium bundled FindBadConstructs plugin ( here: https://chromium.googlesource.com/chromium/src/tools/clang/+/refs/heads/main/plugins )

Reply:
I'm just adding where this flag comes from , people can look up the definition in the original file.